smithy-aws-protocol-tests/model/restJson1/empty-input-output.smithy

```diff
@@ -83,20 +83,28 @@ apply EmptyInputAndEmptyOutput @httpRequestTests([
         protocol: restJson1,
         method: "POST",
         uri: "/EmptyInputAndEmptyOutput",
-        body: "{}",
+        body: "",
         bodyMediaType: "application/json"
     },
 ])
 
 apply EmptyInputAndEmptyOutput @httpResponseTests([
     {
         id: "RestJsonEmptyInputAndEmptyOutput",
-        documentation: "Empty output serializes no payload",
+        documentation: "Empty output deserializes no payload",
         protocol: restJson1,
         code: 200,
         body: "",
         bodyMediaType: "application/json"
     },
+    {
+        id: "RestJsonEmptyInputAndEmptyJSONObjectOutput",
```

Comment:
Nit: we use "Json" everywhere else in these tests, so "RestJsonEmptyInputAndEmptyJsonObjectOutput"